submissions/evgeniy241984/js-dom/script.js

```diff
@@ -0,0 +1,129 @@
+const container = document.querySelector(".container");
+const nav = document.querySelector(".menu");
+const content = document.createElement("div");
+
+function createMainDescription(elem) {
+  nav.after(elem);
+  elem.classList.add("description");
+
+  const descriptionHeader = document.createElement("h2");
+  descriptionHeader.classList.add("description__header");
+  descriptionHeader.textContent = "Star Wars";
+  elem.appendChild(descriptionHeader);
+
+  const divForImg = document.createElement("div");
+  divForImg.classList.add("description__img-container");
+  elem.appendChild(divForImg);
+
+  const img = document.createElement("img");
+  img.setAttribute("src", "images/starWarsSaga.jpg");
+  img.setAttribute("alt", "star wars");
+  divForImg.appendChild(img);
+
+  const divForText = document.createElement("div");
+  divForText.classList.add("description__text-container");
+  elem.appendChild(divForText);
+
+  const paragForText = document.createElement("p");
+  paragForText.textContent =
+    "Star Wars is an American epic space-opera media franchise created by George Lucas, which began with the eponymous 1977 film and quickly became a worldwide pop-culture phenomenon. The franchise has been expanded into various films and other media, including television series, video games, novels, comic books, theme park attractions, and themed areas, comprising an all-encompassing fictional universe.";
+  divForText.append(paragForText);
+}
+createMainDescription(content);
+
+function filmPatter(filmList, item) {
+  const filmHeader = item.querySelector("h2");
+  filmHeader.textContent = filmList.header;
+
+  const filmImage = item.querySelector("img");
+  filmImage.setAttribute("src", filmList.image);
+  filmImage.setAttribute("alt", filmList.alt);
+
+  const filmText = item.querySelector("p");
+  filmText.textContent = filmList.text;
+}
+
+nav.addEventListener("click", filmGenerator);
+
+function filmGenerator({ target }) {
+  let filmNameBtn = target.textContent;
+  listOfFilms.forEach(film => {
+    if (film.header === filmNameBtn) {
+      filmPatter(film, content);
+    }
+  });
```

Comment:
Searching the corresponding object by link content is not a good practice. Take a look at [data-attribute](https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes)
also, read about [Array.prototype.find()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find),
and you can write less code with [Template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)
Using them or not it's up to you.